pkg/ccl/backupccl/system_schema.go

```diff
@@ -814,6 +814,10 @@ var systemTableBackupConfiguration = map[string]systemBackupConfiguration{
 	systemschema.TransactionExecInsightsTable.GetName(): {
 		shouldIncludeInClusterBackup: optOutOfClusterBackup,
 	},
+	systemschema.PlanHintsTable.GetName(): {
+		shouldIncludeInClusterBackup: optInToClusterBackup, // No desc ID columns.
+		customRestoreFunc:            defaultSystemTableRestoreFunc,
```

Comment:
Do we think we'll want to backup/restore plan hints that are applicable to queries that hit a particular table or tables in a particular database when we backup and restore that table or database, even if they're not backing up/restoring the whole cluster?